package/test-package.sh

```diff
@@ -39,8 +39,9 @@ kevm kompile tests/specs/examples/erc20-spec.md                 \
     --main-module VERIFICATION                                  \
     --syntax-module VERIFICATION                                \
     --verbose
-# This test is probably too long for public Github runner and currently seems broken
-if ! ${NIX:-false}; then
-kevm prove tests/specs/examples/erc20-spec.md --backend haskell --format-failures  \
-    --definition tests/specs/examples/erc20-spec/haskell
-fi
+kevm prove tests/specs/examples/erc20-spec.md --backend haskell --definition tests/specs/examples/erc20-spec/haskell
+
+# Commented because of https://github.com/foundry-rs/foundry/issues/545, and we can't install solc 0.8.13
+# forge build --root tests/foundry --no-auto-detect
+# kevm foundry-kompile --foundry-project-root tests/foundry --verbose
+# kevm foundry-prove --foundry-project-root tests/foundry --verbose --test AssertTest.test_assert_true_branch
```

Comment:
Unfortunately, we can't easily enable testing foundry integration on CI without upgrading to `solc` 0.8.19. Seems due to: https://github.com/foundry-rs/foundry/issues/545.